collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -1332,3 +1358,35 @@ def create_lines(slot, lines):
     definitions[chart_name] = {
         'options': [name, title + ': ' + slot_name, units, 'replication slot files', context, chart_type],
         'lines': create_lines(slot_name, chart_template['lines'])}
+
+def add_stat_replications_chart(s,order, definitions, name, secondaries):
+    def create_lines(lines, secondaries):
+        result = list()
+        for line in lines:
+            for secondary in secondaries:
+                secondary=secondary.replace("-","")
+                metric='_'.join([secondary, line[0]])  
+                METRICS[QUERY_NAME_STAT_REPLICATION].append(metric)
+                new_line = [metric, secondary] + line[2:]
+                result.append(new_line)
+            return result
+
+    chart_template = CHARTS[name]
+    position = order.index('database_size')
+    order.insert(position, name)
+    _, title, units, _, context, chart_type = chart_template['options']
+    definitions[name] = {
+        'options': [name, title, units, 'stat replications', context, chart_type],
+        'lines': create_lines(chart_template['lines'], secondaries)}
+
+def CREATE_STAT_QUERY(secondaries):
```

Comment:
```suggestion
def CREATE_STAT_QUERY(secondaries):
```

I suggest make the function name lowercase